ide-config/src/main/resources/eclipse-format.xml

```diff
@@ -0,0 +1,322 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="15">
+<profile kind="CodeFormatterProfile" name="Quarkus" version="15">
```

Comment:
Probably rename to Fabric8 Kubernetes Client

Reply:
sure 🤦  done 🙂